dart/test/sentry_browser_test.dart

```diff
@@ -2,17 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 @TestOn('browser')
-import 'package:sentry/browser.dart';
 import 'package:sentry/sentry.dart';
 import 'package:test/test.dart';
 
 import 'test_utils.dart';
 
 void main() {
-  group('SentryBrowserClient', () {
+  group('SentryClient on browser', () {
     test('SentryClient constructor build browser client', () {
       final client = SentryClient(SentryOptions(dsn: testDsn));
-      expect(client is SentryBrowserClient, isTrue);
+      expect(client is SentryClient, isTrue);
     });
```

Comment:
does this test and the flag `isWeb` makes sense now? or still useful?

Reply:
✅  These client tests were useless indeed, but the flag `isWeb` seems necessary to run test_utils both in browser and VM modes.